servant-rawm-client/servant-rawm-client.cabal

```diff
@@ -21,7 +21,7 @@ library
   exposed-modules:     Servant.RawM.Client
   build-depends:       base >= 4.8 && < 5
                      , servant-rawm >= 1.0
-                     , servant-client-core >= 0.16
+                     , servant-client-core >= 0.16 && < 0.19
```

Comment:
This upper bound is not strictly needed but I figured it could be interpreted as "we tried with 0.18.* and it worked. 
Let me know if I should remove this bound.